Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -191,13 +191,37 @@ class BrowserViewController: UIViewController {
         scrollController.showToolbars(animated: true)
     }
 
+    func SELappWillResignActiveNotification() {
+        // If we are displying a private tab, hide any elements in the browser that we wouldn't want shown 
+        // when the app is in the home switcher
+        guard let privateTab = tabManager.selectedTab where privateTab.isPrivate else {
+            return
+        }
+
+        privateTab.webView?.alpha = 0
+        urlBar.locationView.alpha = 0
+    }
+
+    func SELappDidBecomeActiveNotification() {
+        // Re-show any components that might have been hidden because they were being displayed
+        // as part of a private mode tab
+        UIView.animateWithDuration(0.2, delay: 0, options: UIViewAnimationOptions.CurveEaseInOut, animations: {
+            self.tabManager.selectedTab?.webView?.alpha = 1
```

Comment:
This makes me nervous since the selected tab could change while we're in the background, and then the alpha would never get reset on the original `webView`.

How about we replace `.selectedTab?.webView` with `webViewContainer`? Then all web views will be hidden and we don't need to worry about this getting out of sync.